package.json

```diff
@@ -1,39 +1,45 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/shaunluttin/aurelia-open-id-connect.git"
   },
   "scripts": {
-    "build": "tsc",
-    "lint": "tslint --project tsconfig.json",
-    "package:clean": "rimraf dist",
-    "package:html": "cpx src/**/*.html dist/amd",
-    "package:tsc": "tsc",
-    "package": "npm run package:clean && npm run package:tsc && npm run package:html",
-    "release": "npm run package && git add dist && git commit -m \"prepare release\"",
-    "test": "karma start"
-  },
-  "scripts-comments": {
-    "publish:step:01": "npm run release",
-    "publish:step:02": "npm version <major | minor | patch>",
-    "publish:step:03": "npm publish .",
-    "publish:step:04": "git push --follow-tags"
-  },
-  "jspm": {
-    "main": "index.js",
-    "directories": {
-      "lib": "dist/amd"
-    },
-    "format": "amd",
-    "registry": "npm"
+    "lint": "cross-env tslint --project tsconfig.json",
+    "pretest": "cross-env npm run lint",
+    "test": "cross-env rimraf dist && tsc && karma start --single-run",
+    "develop": "concurrently \"./node_modules/.bin/tsc --watch\" \"./node_modules/.bin/karma start\"",
+    "prebuild:amd": "cross-env rimraf dist/amd",
+    "build:amd": "cross-env tsc --project tsconfig.json --outDir dist/amd --module amd",
+    "postbuild:amd": "cross-env copyfiles --up 1 src/**/*.html src/**/*.css dist/amd",
+    "prebuild:commonjs": "cross-env rimraf dist/commonjs",
+    "build:commonjs": "cross-env tsc --project tsconfig.json --outDir dist/commonjs --module commonjs",
+    "postbuild:commonjs": "cross-env copyfiles --up 1 src/**/*.html src/**/*.css dist/commonjs",
+    "prebuild:es2015": "cross-env rimraf dist/es2015",
+    "build:es2015": "cross-env tsc --project tsconfig.json --outDir dist/es2015 --module es2015 --target es2015",
+    "postbuild:es2015": "cross-env copyfiles --up 1 src/**/*.html src/**/*.css dist/es2015",
+    "prebuild:native-modules": "cross-env rimraf dist/native-modules",
+    "build:native-modules": "cross-env tsc --project tsconfig.json --outDir dist/native-modules --module es2015",
+    "postbuild:native-modules": "cross-env copyfiles --up 1 src/**/*.html src/**/*.css dist/native-modules",
+    "prebuild:system": "cross-env rimraf dist/system",
+    "build:system": "cross-env tsc --project tsconfig.json --outDir dist/system --module system",
+    "postbuild:system": "cross-env copyfiles --up 1 src/**/*.html src/**/*.css dist/system",
+    "prebuild": "cross-env rimraf dist",
+    "build": "concurrently \"npm run build:amd\" \"npm run build:commonjs\" \"npm run build:es2015\" \"npm run build:native-modules\" \"npm run build:system\"",
+    "changelog": "cross-env conventional-changelog -p aurelia-open-id-connect -i doc/CHANGELOG.md -s",
+    "bump-version": "npm --no-git-tag-version version",
+    "preprepare-release": "cross-env npm run test",
+    "prepare-release": "cross-env npm run changelog && npm run build"
   },
-  "license": "MIT",
```

Comment:
Are you not a fan of the MIT license?

Reply:
did not intended this

Reply:
Looks like I moved the MIT license to the top of this file.
It is still present.